tests/deriver/disabled_test_consumer.py

```
@@ -0,0 +1,574 @@
+"""Tests for the consumer module and message processing functionality."""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)